### DISCUSSION TITLE
Start transcription on a specific moment of the video/audio

### DISCUSSION THREAD

**Discussion by JeremiasB23** [openai/whisper#1050] [Q&A] [answered]
Hi, sorry if this question was already made but i can't find anything about this topic. I have an 1 hour long video but the voice starts at minute 10, can I start the transcription at minute 10 with some flag similar to the ones used for  "--language", is this even possible?. Thanks.

**Comment by glangford** [comment, marked as answer]
See the discussion in
* https://github.com/openai/whisper/discussions/755

**Reply by JeremiasB23** [reply]
thank you very much
